pkg/daemon/ceph/client/pool.go

```diff
@@ -327,6 +328,22 @@ func SetPoolProperty(context *clusterd.Context, clusterName, name, propName stri
 	return nil
 }
 
+// SetPoolReplicatedSizeProperty sets the replica size of a pool
+func SetPoolReplicatedSizeProperty(context *clusterd.Context, clusterName, poolName, propVal string, allowUnsafeSingleReplica bool) error {
+	propName := "size"
+	args := []string{"osd", "pool", "set", poolName, propName, propVal}
+	if allowUnsafeSingleReplica && propVal == "1" {
```

Comment:
Since there is validation elsewhere, we won't ever get here unless `allowUnsafeSingleReplica` is true, right? Do we even need to check the flag if there is validation elsewhere?